src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.ts

```diff
@@ -1,97 +1,110 @@
-import { Component, EventEmitter, Input, OnChanges, OnDestroy, Output, SimpleChanges } from '@angular/core';
-import { UntypedFormBuilder } from '@angular/forms';
-import { merge, Subject, Subscription } from 'rxjs';
-import { GroupPermissions } from 'src/app/shared/http-services/group-permissions.service';
-import { ItemEditPerm } from 'src/app/shared/models/domain/item-edit-permission';
-import { ItemGrantViewPerm } from 'src/app/shared/models/domain/item-grant-view-permission';
+import { Component, EventEmitter, Input, OnChanges, OnDestroy, Output, ViewChild } from '@angular/core';
 import { ItemCorePerm } from 'src/app/shared/models/domain/item-permissions';
-import { ItemViewPerm } from 'src/app/shared/models/domain/item-view-permission';
-import { ItemWatchPerm } from 'src/app/shared/models/domain/item-watch-permission';
-import { permissionsConstraintsValidator } from '../../helpers/permissions-constraints-validator';
-import { PermissionsDialogData, generateValues } from '../../helpers/permissions-texts';
-import { TypeFilter } from '../composition-filter/composition-filter.component';
+import { RawGroupRoute } from '../../../../shared/routing/group-route';
+import { GroupPermissions, GroupPermissionsService } from '../../../../shared/http-services/group-permissions.service';
+import { ReplaySubject, switchMap } from 'rxjs';
+import { map, shareReplay } from 'rxjs/operators';
+import { ActionFeedbackService } from '../../../../shared/services/action-feedback.service';
+import { HttpErrorResponse } from '@angular/common/http';
+import { TypeFilter } from '../../helpers/composition-filter';
+import { PermissionsEditFormComponent } from '../permissions-edit-dialog-form/permissions-edit-form.component';
+import { mapToFetchState } from '../../../../shared/operators/state';
 
 @Component({
-  selector: 'alg-permissions-edit-dialog[giverPermissions]',
+  selector: 'alg-permissions-edit-dialog[currentUserPermissions][item][group]',
   templateUrl: './permissions-edit-dialog.component.html',
   styleUrls: [ './permissions-edit-dialog.component.scss' ]
 })
-export class PermissionsEditDialogComponent implements OnChanges, OnDestroy {
+export class PermissionsEditDialogComponent implements OnDestroy, OnChanges {
+  @Output() close = new EventEmitter<boolean>();
 
-  @Input() visible?: boolean;
-  @Input() title?: string;
-  @Input() permissions?: Omit<GroupPermissions,'canEnterFrom'|'canEnterUntil'>;
-  @Input() giverPermissions!: ItemCorePerm;
-  @Input() targetType: TypeFilter = 'Users';
-  @Output() close = new EventEmitter<void>();
-  @Output() save = new EventEmitter<Partial<GroupPermissions>>();
+  @Input() visible = true;
+  @Input() currentUserPermissions!: ItemCorePerm;
+  @Input() item!: { id: string, string: { title: string | null } };
+  @Input() group!: RawGroupRoute;
+  @Input() sourceGroup?: RawGroupRoute;
 
-  targetTypeString = '';
+  @ViewChild(PermissionsEditFormComponent) permissionsEditForm?: PermissionsEditFormComponent;
```

Comment:
it is still there (cfr https://github.com/France-ioi/AlgoreaFrontend/pull/1132/files#r963471956)